drivers/spi/spi_ll_stm32.c

```diff
@@ -747,6 +741,18 @@ static int transceive_dma(const struct device *dev,
 	/* This is turned off in spi_stm32_complete(). */
 	spi_stm32_cs_control(dev, true);
 
+#if defined(CONFIG_SPI_STM32_INTERRUPT) || defined(CONFIG_SPI_ASYNC)
```

Comment:
Do we want to change behavior in config Interrup + DMA (SPI_ASYNC = n) ?
Isn't it sufficient to do `defined(CONFIG_SPI_ASYNC)` here ?

Reply:
Do you use DMA in this case? It looks like `transceive` function in async mode without DMA.

Reply:
> Do you use DMA in this case? It looks like `transceive` function in async mode without DMA.

yes, if `CONFIG_SPI_STM32_DMA=y`